examples/angular/src/app/components/content/content.component.ts

```diff
@@ -59,7 +59,7 @@ export class ContentComponent implements OnInit, OnDestroy {
   }
 
   signIn() {
-    this.selector.show();
+    // this.selector.show();
```

Comment:
How should these be handled now?

Reply:
We do it by getting the reference of the "wallet-selector-modal" similar like in react example, I will updatethe angular example too.